README.md

```diff
@@ -1,5 +1,5 @@
 [![Build Status](https://travis-ci.org/mmagnuski/borsar.svg?branch=master)](https://travis-ci.org/mmagnuski/borsar)
-[![Coverage Status](https://codecov.io/gh/mmagnuski/borsar/branch/master/graph/badge.svg)](https://codecov.io/gh/mmagnuski/borsar)   
+[![Coverage Status](https://codecov.io/gh/mmagnuski/borsar/branch/master/graph/badge.svg)](https://codecov.io/gh/mmagnuski/borsar)
```

Comment:
I think that by removing these spaces you remove a newline, correct? Some editors remove trailing whitespace, but for GitHub markdown these whitespaces do change how the content is displayed.

Reply:
No, in this case it's just unneeded whitespace, I think. The badges are all on one line currently as well: 
![image](https://user-images.githubusercontent.com/9084751/167361130-2054a4f7-d9c5-4b3f-af25-e51da73d584f.png)

I think to get a new line in markdown, you need an empty line in between lines.

Reply:
Ok, good, I was always using two trailing whitespace, which always works as newline, but it be easier to switch to empty lines, because my editor also eats trailing whitespaces. :)